elastic/kibana
# [Upgrade Assistant] Fix typo in retrieval of cluster settings

Fixes https://github.com/elastic/kibana/issues/116322

**How to test**
1. Start up elasticsearch, `yarn es snapshot --license=trial`
2. Start up Kibana, `yarn start`
3. Navigate to `Stack Management` -> `Upgrade Assistant`
5. Verify that the deprecation logs collection toggle is enabled by default
